.github/workflows/post-build.yaml

```diff
@@ -210,22 +210,63 @@ jobs:
     needs:
       - component-descriptor
       - repo-metadata
-    # only run for default-branch, and never run for forks
-    if: ${{ github.ref_name == github.event.repository.default_branch &&
-            ! fromJSON(needs.repo-metadata.outputs.fork) &&
-            github.event_name == 'push'
-        }}
+    # never run for forks and only for push events
+    if: ${{ ! fromJSON(needs.repo-metadata.outputs.fork) && github.event_name == 'push' }}
     runs-on: ${{ vars.DEFAULT_RUNNER || 'ubuntu-latest' }}
     permissions:
       contents: write
       id-token: write
     steps:
+      - uses: actions/checkout@v4
```

Comment:
I assume you do not use `trusted-checkout`, because this is not required for push-triggered runs?